README.md

```diff
@@ -1,27 +1,29 @@
-# ![Akira](akira-logo-transparent.png)
-> The Linux Design Tool
+<h1 align="center">
+    <br>
+    <img src="akira-logo-transparent.png" alt="Akira">
+</h1>
+<h4 align="center">The Linux Design Tool</h4>
+<p align="center">
+    <a href="https://travis-ci.org/akiraux/Akira"><img src="https://travis-ci.org/akiraux/Akira.svg"
+            alt="Build Status"></a>
+    <a href="https://liberapay.com/AkiraUX"><img
+            src="http://img.shields.io/liberapay/patrons/AkiraUX.svg?logo=liberapay" alt="AkiraUX on Liberapay"></a>
+    <a href="https://www.patreon.com/akiraux"><img
+            src="https://img.shields.io/badge/patreon-donate-orange.svg?logo=patreon" alt="AkiraUX on Patreon"></a>
+</p>
+<p align="center">
+    <a href="#-compile">Compile</a> •
+    <a href="#-questions-issues">Questions</a> •
+    <a href="#-contributing">Contributing</a> •
+    <a href="#-support">Support</a> •
+    <a href="#-mascot">Mascot</a> •
+    <a href="#-license">License</a>
+</p>
+
+![screenshot](akira-screenshot.png)
+
+## 🛠 Compile
 
-[![Build Status](https://travis-ci.org/akiraux/Akira.svg)](https://travis-ci.org/akiraux/Akira) [![AkiraUX on Liberapay](http://img.shields.io/liberapay/patrons/AkiraUX.svg?logo=liberapay)](https://liberapay.com/AkiraUX/)  [![AkiraUX on Patreon](https://img.shields.io/badge/patreon-donate-orange.svg?logo=patreon)](https://www.patreon.com/akiraux)
-
-**AKIRA IS CURRENTLY IN EARLY DEVELOPMENT, NOT READY TO BE USED!**
-
-Akira is a native Linux Design application built in Vala and GTK. Akira focuses on offering a modern and fast approach to UI and UX Design, mainly targeting web designers and graphic designers.
```

Comment:
Let's keep the description please

Reply:
Actually, I forgot to add this description before pushing. Adding it now. 😁